Integrations/VMRay/VMRay.py

```diff
@@ -490,6 +490,56 @@ def get_submission(submission_id):
     response = http_request('GET', url_suffix=suffix)
     return response
 
+def check_file_command():
+    file_hash_input = demisto.args().get('file')
+    file_hashes = file_hash_input.split(',')
```

Comment:
please use `argToList`

Reply:
I haven't modified this from the original code. I will leave this for someone else to look at.